parsely/src/main/java/com/parsely/parselyandroid/FlushQueue.kt

```diff
@@ -33,22 +32,21 @@ internal class FlushQueue(
 
                 val jsonPayload = toParselyEventsPayload(eventsToSend)
                 if (skipSendingEvents) {
-                    log("Debug mode on. Not sending to Parse.ly. Otherwise, would sent ${eventsToSend.size} events: $jsonPayload")
+                    Log.d("Debug mode on. Not sending to Parse.ly. Otherwise, would sent ${eventsToSend.size} events: $jsonPayload")
                     repository.remove(eventsToSend)
                     return@launch
                 }
-                log("Sending request with %d events", eventsToSend.size)
-                log("POST Data %s", jsonPayload)
-                log("Requested %s", ParselyTrackerInternal.ROOT_URL)
+                Log.d("Sending request with ${eventsToSend.size} events")
+                Log.d("POST Data $jsonPayload")
+                Log.d("Requested ${ParselyTrackerInternal.ROOT_URL}")
                 restClient.send(jsonPayload)
                     .fold(
                         onSuccess = {
-                            log("Pixel request success")
+                            Log.i("Pixel request success")
```

Comment:
Question (❓): This is the only `Log.i("Xyz")` that I found. Just for my FYI, why did you chose INFO instead of DEBUG here? 🤔

Reply:
I wanted this log to be kept in the production variants of consumer apps (`DEBUG` level logs are stripped)

Reply:
👍 if that was indeed intentional, I just wasn't sure about it! 😊